app/src/userpages/components/StreamPage/Show/KeyView/index.jsx

```diff
@@ -38,11 +38,14 @@ export class KeyView extends Component<Props> {
         }
     }
 
-    addKey = (key: string) => {
+    addKey = (key: string): Promise<void> => new Promise((resolve, reject) => {
```

Comment:
Hmm, I wonder if this would be cleaner with async/await?

Reply:
I was wondering that does it need to return Promise? I don't see any place that uses `await` or `then` to wait `addKey` completes.